shamil_latypov/graph.cpp

```diff
@@ -142,7 +142,13 @@ int Graph::get_edges_cnt() const {
 
 std::vector<EdgeId> Graph::get_edge_vector_for_color(
     const Edge::Color& color) const {
-  return colored_edges_.at(color);
+  for (const auto& edge : get_edges()) {
+    if (edge.get_color() == color)
+      return colored_edges_.at(color);
+  }
+
+  std::vector<EdgeId> edge_ids;
+  return edge_ids;
 }
```

Comment:
1) :))) Вот ты придумал, тогда уж надо было проверить, есть ли такой ключ в карте:
```cpp
if (colored_edges.find(color) == end()) { . . . }
```

2) Переименуй метод, а то `vector` писать в названии не очень хорошо. Тем более у тебя уже есть хорошее название переменной, которую ты тут используешь
```cpp
get_colored_edges();
```

3) Нет нужды делать копию, верни `const` ссылку

4) В такой ситуации можно написать следующую логику:
```suggestion
const std::vector<EdgeId>& Graph::get_colored_edges(
    const Edge::Color& color) const {
  if (colored_edges_.find(color) == colored_edges_.end()) {
      static std::vector<EdgeId> empty_result;
      return empty_result;
    }
    return colored_edges_.at(color);
}
```